cloud/deploys/bin/azure/bastion.sh

```diff
@@ -20,11 +19,10 @@ function bastion::get_vm_ip() {
 #   2: the name of the database
 #######################################
 function bastion::get_postgres_host() {
-  readonly POSTGRES_HOST=$(az postgres server show \
-  -g "${1}" \
-  -n "${2}" \
-  --query "fullyQualifiedDomainName" | tr -d '"')
-  echo "${POSTGRES_HOST}"
+  echo $(az postgres server show \
+    -g "${1}" \
+    -n "${2}" \
+    --query "fullyQualifiedDomainName" | tr -d '"')
```

Comment:
question (blocking): is the echo required or could this be
```suggestion
  az postgres server show \
    -g "${1}" \
    -n "${2}" \
    --query "fullyQualifiedDomainName" \
    | tr -d '"'
```

Reply:
ahh yeah the other ones are echo'd then eval'd but they could just be eval'd let me try that